internal/server/testdata/openapi3.json

```diff
@@ -3229,6 +3229,21 @@
             "application/json": {
               "schema": {
                 "properties": {
+                  "api": {
+                    "properties": {
+                      "clientEmail": {
+                        "type": "string"
+                      },
+                      "domainAdmin": {
+                        "type": "string"
+                      },
+                      "privateKey": {
+                        "example": "-----BEGIN PRIVATE KEY-----\nMIIDNTCCAh2gAwIBAgIRALRetnpcTo9O3V2fAK3ix+c\n-----END PRIVATE KEY-----\n",
+                        "type": "string"
+                      }
+                    },
+                    "type": "object"
+                  },
```

Comment:
are we concerned at all about google-specific fields here?

Reply:
It would be nice to qualify these a bit more with another term. In the CLI we used `service-account`. Maybe we could do that here as well with `serviceAccountDomainAdmin`, and `serviceAccountProviateKey` ?

Reply:
Opened #2666 so we can have some proper discussion around re-naming these fields, I'm not a huge fan of Google specific fields in the request object either. I'm leaning towards grouping them in a `google` key.